### PR TITLE
[12.0] [MIG] project_timeline_hr_timesheet

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**
Project Timeline - Timesheet
============================
This module is an addition to the module Project Timeline.
It adds the progress of tasks (hr_timesheet) on the timeline view.

- [x] Depends on https://github.com/OCA/project/pull/466

### CODE REVIEW

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza @OCA/project-service-maintainers Could you please review it?

File: project_timeline_hr_timesheet/views/project_task_view.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<!-- Copyright 2018 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <record id="project_task_timeline" model="ir.ui.view">
+        <field name="name">project_task_timeline</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project_timeline.project_task_timeline"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='user_id']" position="after">
```
Comment (alexey-pelykh) [inline project_timeline_hr_timesheet/views/project_task_view.xml]: `xpath` is an overkill here, `field` would be enough

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

**Comment by alexey-pelykh** [comment]
Depends on #466

**Comment by Nikul-Chaudhary** [comment]
@alexey-pelykh Done

**Comment by alexey-pelykh** [comment]
Will review once `project_timeline` is greenish in runbot

**Comment by rafaelbn** [comment]
I set WIP until dependencies are merged. Thanks for this contribution!

**Comment by pedrobaeza** [comment]
Please rebase

**Comment by Nikul-Chaudhary** [comment]
> Please rebase

rebased

**Review by pedrobaeza — APPROVED** [review]
Squash together the 2 "README.rst" commits.

File: project_timeline_hr_timesheet/static/src/less/project_timeline_hr_timesheet.less [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.o_project_timeline_hr_timesheet_progress {
```
Comment (tarteo) [inline project_timeline_hr_timesheet/static/src/less/project_timeline_hr_timesheet.less]: Use scss instead

**Review by tarteo — CHANGES_REQUESTED** [review]

**Review by emagdalenaC2i — APPROVED** [review]

**Pull request merged**